products/rhel10/profiles/anssi_bp28_enhanced.profile

```diff
@@ -22,32 +22,33 @@ description: |-
 
 selections:
     - anssi:all:enhanced
-    # Following rules are incompatible with the rhel10 product
-    - '!partition_for_opt'
+    - var_password_hashing_algorithm=SHA512
+    - var_password_pam_unix_rounds=65536
```

Comment:
```suggestion
    - var_password_pam_unix_rounds=11
```
In alignment to `yescrypt`

Reply:
I implemented this change in form of 758c5e3f1f725101925e9c7ccb6e498cc1ba0ac7